FileBrowser/FileBrowser.swift

```diff
@@ -62,7 +62,11 @@ open class FileBrowser: UINavigationController {
                                                             showSize: showSize)
 
         self.init(rootViewController: fileListViewController)
-        self.view.backgroundColor = UIColor.white
+        if #available(iOS 13, *) {
+            self.view.backgroundColor = UIColor.systemBackground
+        } else {
+            self.view.backgroundColor = UIColor.white
+        }
```

Comment:
Add UIColor extension with the function:
func whiteBackground -> UIColor {
   if #available(iOS 13, *) {
      return .systemBackground
   } else {
      return .white 
   }
}

Reply:
You're right!
Thanks for the suggestion

Reply:
Good to see you here by the way 🎉